actionlist/commons.js

```diff
@@ -7,7 +7,7 @@ export function getListJSON() {
astApprovedDate\":\"2017-02-09T11:30:00\",\"creationDate\":\"2017-02-09T12:00:00\",\"_links\":{\"self\":{\"href\":\"https://workflow.iu.edu/actions/12345\"},\"actionList\":{\"href\":\"https://workflow.iu.edu/actions\"},\"processInstance\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/open\"},\"log\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/log\"},\"initiator\":{\"href\":\"https://people.iu.edu/fraferna\"}}},{\"id\":\"12352\",\"processInstanceId\":\"23456\",\"processType\":{\"name\":\"SomeProcess\",\"label\":\"Some Process\"},\"title\":\"Hello, world!\",\"processInstanceStatus\":{\"code\":\"R\",\"label\":\"Enroute\"},\"actionRequested\":{\"code\":\"A\",\"label\":\"Approve\"},\"initiator\":\"Francis Fernandez\",\"delegator\":\"Eric Westfall\",\"group\":\"ESI Developers\",\"routeNodes\":[\"Manager Approval\"],\"lastApprovedDate\":\"2017-02-10T11:30:00\",\"creationDate\":\"2017-02-10T12:00:00\",\"_links\":{\"self\":{\"href\":\"https://workflow.iu.edu/actions/12345\"},\"actionList\":{\"href\":\"https://workflow.iu.edu/actions\"},\"processInstance\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/open\"},\"log\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/log\"},\"initiator\":{\"href\":\"https://people.iu.edu/fraferna\"}}},{\"id\":\"12353\",\"processInstanceId\":\"23456\",\"processType\":{\"name\":\"SomeProcess\",\"label\":\"Some Process\"},\"title\":\"Hello, world!\",\"processInstanceStatus\":{\"code\":\"R\",\"label\":\"Enroute\"},\"actionRequested\":{\"code\":\"A\",\"label\":\"Approve\"},\"initiator\":\"Francis Fernandez\",\"delegator\":\"Eric Westfall\",\"group\":\"ESI Developers\",\"routeNodes\":[\"Manager Approval\"],\"lastApprovedDate\":\"2017-02-11T11:30:00\",\"creationDate\":\"2017-02-11T12:00:00\",\"_links\":{\"self\":{\"href\":\"https://workflow.iu.edu/actions/12345\"},\"actionList\":{\"href\":\"https://workflow.iu.edu/actions\"},\"processInstance\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/open\"},\"log\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/log\"},\"initiator\":{\"href\":\"https://people.iu.edu/fraferna\"}}},{\"id\":\"12354\",\"processInstanceId\":\"23456\",\"processType\":{\"name\":\"SomeProcess\",\"label\":\"Some Process\"},\"title\":\"Hello, world!\",\"processInstanceStatus\":{\"code\":\"R\",\"label\":\"Enroute\"},\"actionRequested\":{\"code\":\"A\",\"label\":\"Approve\"},\"initiator\":\"Francis Fernandez\",\"delegator\":\"Eric Westfall\",\"group\":\"ESI Developers\",\"routeNodes\":[\"Manager Approval\"],\"lastApprovedDate\":\"2017-02-12T11:30:00\",\"creationDate\":\"2017-02-12T12:00:00\",\"_links\":{\"self\":{\"href\":\"https://workflow.iu.edu/actions/12345\"},\"actionList\":{\"href\":\"https://workflow.iu.edu/actions\"},\"processInstance\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/open\"},\"log\":{\"href\":\"https://workflow.iu.edu/processInstances/23456/log\"},\"initiator\":{\"href\":\"https://people.iu.edu/fraferna\"}}},";
```

Comment:
This string shouldn't use double quotes in the outside most quote (you can have double quotes inside single quotes and then you don't have to escape them (e.g., `const str = '{"id": "12345"}';`) and it shouldn't break parsing JSON in a decent parser. It looks like the rule that checks for that violation was disabled for some reason (the same change should be made on the `str` constant in `getListJSON()`).